manifests/weave/kissnet/bns/bns-faucet-patch.yaml

```diff
@@ -0,0 +1,4 @@
+# patch examples here: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md
+- op: replace
+  path: /spec/template/spec/containers/0/env/3/value
```

Comment:
Does this translate to the `FAUCET_CREDIT_AMOUNT_IOV` env variable?

Reply:
yep, it is a patch for the base file, env vars are a list, so yep, it's a list sequence.